proto/viam/robot/v1/robot.proto

```diff
@@ -77,17 +77,21 @@ service RobotService {
   }
 
   // GetStatus returns the list of all statuses requested. An empty request signifies all resources.
+  // This service is deprecated
```

Comment:
```suggestion
  // This method is deprecated.
```

or we can just remove since adding `deprecated = true` marks this pretty clearly in docs. 

![image](https://github.com/viamrobotics/api/assets/7954103/657d2fb7-4390-4eed-a25a-eaddc473f5e1)

```suggestion
```